Updated Models: Role, Permission, & Team

metaversedataman opened the pull request:
## Added fillable field to Models: Role, Permission, & Team

Solved the bug on using the below:

```
Role::create
Permission::create
Team::create
```

which throws error: add "name" to fillable.

santigarcor commented:
Here I only see you modified Role, Permission and Team are missing that modification

metaversedataman commented:
Sorry about that. I thought the commit was on the 3 files.
I have created 2 additional pull requests for Permission & Team.
Thank you.

metaversedataman commented:
Fixed!

I deleted other patches, and added the changes in the 2 models: Permission.php & Team.php, both in path-1; as you can see in the commits above. This was exactly what I had wanted to do before, but ended up creating multiple patches.

Thank you.
